pkg/kube/wait.go

```diff
@@ -114,6 +115,14 @@ func (w *waiter) waitForResources(created ResourceList) error {
 				if !w.deploymentReady(newReplicaSet, currentDeployment) {
 					return false, nil
 				}
+			case *storagev1beta1.StorageClass:
```

Comment:
The storage APIs are now at `v1`, so we should make sure to have a case for that api version as well and also just use the v1 storage client to fetch both of them (like what we do with deployments)

Reply:
I have updated it to stroragev1